packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/contexts/DiagramDescriptionContext.types.ts

```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { GQLDiagramDescription } from '../representation/DiagramRepresentation.types';
+
+export interface DiagramDescriptionContextValue {
+  diagramDescription: GQLDiagramDescription | null;
```

Comment:
We can guarantee that it will never be null while using it inside of our diagram since we won't display diagrams until we have receive this.